### PR TITLE
Add Sonic Screwdriver to recipes.

### PR DESCRIPTION
**Pull request by AjaxIronside**

### CODE REVIEW

File: src/recipes/sonic-screwdriver.JSON [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        "1 oz vodka",
+        "6 oz. Sprite"
+        "Ice cubes"
+    ],
+    "directions": [
+        "Fill a 12 oz glass halfway with ice.",
+        "Mix together soda and alcohol and pour over ice.",
+        "Enjoy with your favorite companion."
+    ],
+    "image": "<sonic-screwdriver.jpg>"
```
Comment (alfg) [inline src/recipes/sonic-screwdriver.JSON]:
```suggestion
    "image": "sonic-screwdriver.jpg"
```

Comment (AjaxIronside) [reply src/recipes/sonic-screwdriver.JSON]: Requested changes have been made. Thanks for your feedback!

File: src/recipes/sonic-screwdriver.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Sonic Screwdriver",
+    "description": "Like a Screwdriver, with a little Timey-Wimey.",
+    "github": "AjaxIronside",
+    "ingredients": [
+        "1.5 oz Blue Curaçao.",
+        "1 oz vodka",
+        "6 oz. Sprite"
```
Comment (alfg) [inline src/recipes/sonic-screwdriver.json]:
```suggestion
        "6 oz. Sprite,"
```

**Comment by alfg** [comment]
Awesome, thanks!

**Pull request merged**